### DISCUSSION TITLE
How Min renders the page?

### DISCUSSION THREAD

**Discussion by PowerzinBR** [minbrowser/min#2418] [General]
Hey there! I was a little bit curious to know how min renders the page, i mean, there's an <webview> and a <iframe>, and i didn't find anything related, thanks also for the answer.

Hope everyone is good! 😄

**Comment by PalmerAL** [comment]
Min is based on [Electron](https://www.electronjs.org/); in particular there's a [BrowserView](https://www.electronjs.org/docs/latest/api/browser-view) API that allows for loading webpages inside a window.

**Comment by PowerzinBR** [comment]
Ohhhh, i understand it. Thanks for telling me
